### DISCUSSION TITLE
Custom Satellite positions

### DISCUSSION THREAD

**Discussion by willichan** [GitSquared/edex-ui#1105] [Ideas]
I would like to have the ability to set the satellite positions (CSV with longitude, latitude, and elevation) to be over major corporate facilities or points of interest are located around the globe.

**Comment by ghost** [comment]
If you build from source, you can just push those onto the constellation array [here](https://github.com/GitSquared/edex-ui/blob/master/src/classes/locationGlobe.class.js#L107).
